game_parser/parser_app/parser.py

```diff
@@ -0,0 +1,43 @@
+from igdb.wrapper import IGDBWrapper
+from .models import Games
+
+import json
+import requests
+
+with open('.secrets', 'r') as secrets:
+    secrets_json = json.load(secrets)
+
+CLIENT_ID = secrets_json['client_id']
+SECRET_CODE = secrets_json['secret_code']
+
+wrapper = IGDBWrapper(client_id=CLIENT_ID, auth_token=SECRET_CODE)
+
+token_response = requests.request("POST",
+                                  f"https://id.twitch.tv/oauth2/token?client_id={CLIENT_ID}&client_secret={SECRET_CODE}&"
+                                  f"grant_type=client_credentials").json()
+
+access_token = token_response['access_token']
+
+headers = {
+    'Client-ID': CLIENT_ID,
+    'Authorization': f"Bearer {access_token}"
+}
+
+body_games = 'fields *;'
+
+
+def parse_games() -> list:
```

Comment:
```suggestion
def parse_games() -> List:
```
For the typehinting we use types from the `typing` lib

Reply:
Also if you decided to use typehints here then add type of the items from that list
Would looks smth like `List[Dict[str, Any]]`

Reply:
Technically you're not parsing games here, but firstly request them from igdb and then parsing follows 
That's why `get_games` name for that method fits better

Reply:
Done